packages/web-components/src/text/text.ts

```diff
@@ -108,4 +158,55 @@ export class Text extends FASTElement {
    */
   @attr
   align?: TextAlign;
+
+  /**
+   * Handles changes to align attribute custom states
+   * @param prev - the previous state
+   * @param next - the next state
+   */
+  public alignChanged(prev: TextAlign | undefined, next: TextAlign | undefined) {
+    if (prev) {
+      toggleState(this.elementInternals, prev, false);
+    }
+    if (next) {
+      toggleState(this.elementInternals, next, true);
+    }
+  }
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+
+    Observable.getNotifier(this).subscribe(this);
```

Comment:
I assume this is more performant than all the `[attribute]Changed` observers?

Reply:
It's basically an efficiency given all the instances I'm using them for are boolean attributes, so all that code would be duplicated. If any of those change, handleChange is called for it and it should fire the method to toggle the state. Reduces the duplication for each and moves them into a single observable function for handleChange